bedrock/foundation/templates/foundation/base-resp.html

```diff
@@ -44,8 +54,32 @@
         </li>
       {% endfor %}
       </ul>
+      <ul class="nav">
```

Comment:
I didn't like this either but the first set of side navigation markup was already built like this and I didn't want to mess with it; I just copied what was there. Since this is somewhat time sensitive, can I follow up with this in another PR?